translations/messages-en.properties

```diff
@@ -971,3 +971,6 @@ Submit\ via\ sms = SMS only
 Submit\ via\ either = Try internet first, then SMS
 place.deleted = [deleted]
 place.unavailable = [unavailable]
+contact.deceased = Deceased
+contact.died = Died
```

Comment:
This looks like it would be hard to translate without clearer keys: died and deceased mean the same thing depending on the context they're in.

Reply:
Fixed